application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/common/DurationHdrHistogram.java

```diff
@@ -101,11 +103,13 @@ public IQuantity getDurationAtPercentile(IQuantity percentile) {
 	}
 
 	/**
-	 * Computes the duration at a given percentile for values stored
-	 * in the histogram, and number of values at or above that duration.
-	 * @param percentile - the percentile, as a {@link UnitLookup#NUMBER}
-	 * @return a pair with the computed duration as a {@link UnitLookup#TIMESPAN},
-	 * 	       and item count as a {@link UnitLookup#NUMBER}, in that order
+	 * Computes the duration at a given percentile for values stored in the histogram, and number of
+	 * values at or above that duration.
+	 * 
+	 * @param percentile
+	 *            - the percentile, as a {@link UnitLookup#NUMBER}
```

Comment:
Do we want to remove the hyphen here?

Reply:
Done! (Not that it's really part of this work, but always leave the camping ground cleaner. ;) )